README.md

```diff
@@ -56,6 +56,39 @@ Once you are ready, you can save your file:
 pipeline.save(pipeline.get_derivative_path(bids_path), overwrite=True)
 ```
 
+## ▶️ Example HPC Environment Setup
+
+Assuming you are on a system such as [Narval](https://docs.alliancecan.ca/wiki/Narval/en):
+
+```bash
+module load python/3.10
+
+# Build the virtualenv in your homedir
+virtualenv --no-download eeg-env
+source eeg-env/bin/activate
+
+pip install --no-index mne
+pip install --no-index pandas
+pip install --no-index xarray
+pip install --no-index pyyaml
+pip install --no-index sklearn
+pip install mne_bids
+
+# Clone down mne-iclabel and switch to the right version and install it locally
+git clone https://github.com/mne-tools/mne-icalabel.git
```

Comment:
Out of curiosity - is it necessary to checkout a certain branch of mne-icalabel?